Trading/Mode/daily_trading_mode/daily_trading.py

```diff
@@ -548,10 +551,20 @@ async def _create_order(
                 params.update(param_update)
                 chained_orders.append(chained_order)
         if len(chained_orders) > 1:
-            stop_count = len([o for o in chained_orders if trading_personal_data.is_stop_order(o.order_type)])
-            tp_count = len(chained_orders) - stop_count
-            group_type = trading_personal_data.OneCancelsTheOtherOrderGroup if stop_count == tp_count \
-                else trading_personal_data.BalancedTakeProfitAndStopOrderGroup
+            stop_orders = [o for o in chained_orders if trading_personal_data.is_stop_order(o.order_type)]
+            tp_orders = [o for o in chained_orders if not trading_personal_data.is_stop_order(o.order_type)]
+            if len(stop_orders) == len(tp_orders):
+                group_type = trading_personal_data.OneCancelsTheOtherOrderGroup
+            elif trailing_profile_type == trading_personal_data.TrailingProfileTypes.FILLED_TAKE_PROFIT:
+                group_type = trading_personal_data.TrailingOnFilledTPBalancedOrderGroup
+                entry_price = current_order.origin_price
+                for stop_order in stop_orders:
+                    # register trailing profile in stop orders
+                    stop_order.trailing_profile = trading_personal_data.create_filled_take_profit_trailing_profile(
```

Comment:
this initializes a FILLED_TAKE_PROFIT trailing profile bound to the group take profits prices and entry price